### DISCUSSION TITLE
Center content horizontally

### DISCUSSION THREAD

**Discussion by sduras** [mmistakes/jekyll-theme-basically-basic#135] [Q&A] [answered]
Please advise how to center page content horizontally on desktop and larger viewports. Currently, it is left-aligned, and the right side of page is empty. Thank you in advance.

**Comment by motilalmeher** [comment, marked as answer]
follow this 
https://github.com/mmistakes/jekyll-theme-basically-basic/issues/98

**Reply by sduras** [reply]
Thank you.
